ecl/hql/hqlgram.y

```diff
@@ -1742,6 +1743,11 @@ persistOpt
     : fewMany
     | expireAttr
     | clusterAttr
+    | REFRESH '(' expression ')'
```

Comment:
It needs to be a compile time constant, because the information about whether to refresh or not is stored as a constant in the workunit.  If it was changed to call a function then it could be dynamic - but that would also complicate the implementation, and I doubt it would be used.

You could use constExpression for clarity, not much in it really.

The format changing is covered by the checks that are already in place when reading a disk file.